seqr/views/utils/test_utils.py

```diff
@@ -766,6 +768,7 @@ def call_request_json(self, index=-1):
         'chrom': '2',
         'clinvar': {'clinicalSignificance': None, 'alleleId': None, 'variationId': None, 'goldStars': None},
         'familyGuids': ['F000002_2', 'F000003_3'],
+        'cpxIntervals': [],
```

Comment:
I would actually update the fixture data for at least one variant to make sure the parsing for this field works right